src/evaluations/analyzer.py

```diff
@@ -278,6 +282,60 @@ def __init__(self, out_dir, evaluation_directory, evaluation_run_name,
     super().__init__(out_dir, evaluation_directory, evaluation_run_name,
                      evaluation_name, estimable_criteria_list, plot_params)
 
+  def convert_raw_df_to_long_format(self):
+    """Convert the raw DataFrame to a long format.
```

Comment:
Should probably describe what the long format is somewhere in the doc string.

Reply:
Done.